### PR TITLE
net/vnstat2: Add to repo

### PR DESCRIPTION
**Pull request by diizzyy**
Maintainer: Community / Anyone
Compile tested: ar71xx, TP-Link TL-WDR3600, OpenWrt master
Run tested: ar71xx, TP-Link TL-WDR3600, OpenWrt master

Description:
Add vnstat 2.X to repo

Signed-off-by: Daniel Engberg <daniel.engberg.lists@pyret.net>

### CODE REVIEW

File: net/vnstat2/Makefile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=vnstat2
+PKG_VERSION:=2.2
+PKG_RELEASE:=1
+PKG_MAINTAINER:=
```
Comment (neheb) [inline net/vnstat2/Makefile]: For consistency, I'd put these after PKG_BUILD_DIR. Why leave maintainer blank?

File: net/vnstat2/Makefile [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE:=vnstat-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://humdi.net/vnstat/
+PKG_HASH:=c60a7bc35e0247b0d2e628d8fc39427f2482e844c2c7b9cdbfc814463310e02c
+PKG_BUILD_DIR:=$(BUILD_DIR)/vnstat-$(PKG_VERSION)
+
+PKG_INSTALL:=1
```
Comment (neheb) [inline net/vnstat2/Makefile]: Maybe PKG_BUILD_PARALLEL could be used.

File: net/vnstat2/Makefile [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  ensures light use of system resources regardless of network
+  traffic rate. 
+endef
+
+define Package/vnstat2/conffiles
+/etc/vnstat.conf
+endef
+
+TARGET_CFLAGS += -ffunction-sections -fdata-sections -flto
+TARGET_LDFLAGS += -Wl,--gc-sections -Wl,--as-needed -flto
```
Comment (neheb) [inline net/vnstat2/Makefile]: Could be rewritten as -Wl,--gc-sections,--as-needed

File: net/vnstat2/Makefile (hunk 1 above)
Comment (diizzyy) [reply net/vnstat2/Makefile]: I'd prefer it to be a community package however if a maintainer is needed I'll drop the PR unless someone wants to step up.

File: net/vnstat2/Makefile (hunk 2 above)
Comment (diizzyy) [reply net/vnstat2/Makefile]: I'll see if I can test it somehow, it's a really small package so I don't think there's much to gain however...

**Comment by diizzyy** [comment]
Nevermind, I'll keep this in my own repo

**Pull request closed without merging**